### ISSUE TITLE
Add TTS Accessibility Support 

### ISSUE THREAD

**Issue by GanstaKingofSA** [GanstaKingofSA/DDLCModTemplate2.0#5]
Feature Set Added when Ready/Complete from Git Repository: https://github.com/therationalpi/DDLC-self-voicing

This feature allows users that cannot see to enjoy DDLC and Mods using a accessibility key in Ren'Py that allows the game to read who is saying the dialogue and what they are saying (e.g. Natsuki says Gross)
May experiment with it myself to get Ren'Py's actual feature enabled by default via a modified Ren'Py file but will add Pi's or whatever comes first.

Issues that may come with patch:
Glitch Text may be said which shouldn't in Act 2-4 (e.g. Monika says j4%6!a6%)
Sound Effects may need to be said (e.g. Natsuki slaps Sayori with a cookie)

**Comment by GanstaKingofSA** [comment]
Pending Response if Feature is Beta or Ready. Moving out of new update when ready

**Comment by GanstaKingofSA** [comment]
Hasn't been a update for over a year with TTS in DDLC. Might bring it back eventually but for now closed.
